bull-common/src/test/java/com/hotels/beans/sample/ExtendedFromFooSimple.java

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
Instead of creating a new class, can you use this existing one: `FromFooSubClass` for your test?

Reply:
This is done.